### PR TITLE
Update table-objects.rst

### PR DESCRIPTION
**Pull request by jeffblack360**
Working my way through this part of the doc I noticed a few things that I thought needed some clarification. Let me know if you spot anything that should be tweaked.

### CODE REVIEW

File: en/orm/table-objects.rst [hunk 1]
```diff
@@ -166,17 +166,17 @@ buildRules
 
 .. php:method:: buildRules(Event $event, RulesChecker $rules)
 
-The ``Model.buildRules`` event is fired before after a rules instance has been
-created and the table's ``beforeRules()`` method has been called.
+The ``Model.buildRules`` event is fired after a rules instance has been
+created and before the table's ``beforeRules()`` method has been called.
```
Comment (lorenzo) [inline en/orm/table-objects.rst]: I think this should read table's `buildRules()` instead

Comment (jeffblack360) [inline en/orm/table-objects.rst]: I'll have to double-check the code, but I thought the order was buildRules, beforeRules, then afterRules.  Thanks!

Comment (markstory) [inline en/orm/table-objects.rst]: The order in the code is:

1. Table::buildRules() is called.
2. Model.buildRules event is triggered.
3. Model.beforeRules event is triggered.
4. Check is done
5. Model.afterRules event is triggered.

Comment (jeffblack360) [inline en/orm/table-objects.rst]: Ok sorry to waffle on opening or closing this PR but the documenation is confusing and I think it's best to clarify the language.

**Pull request merged**